.pyre_configuration

```diff
@@ -4,7 +4,8 @@
     ".*/generate_taint_models/.*",
     ".*/scripts/.*",
     ".*/source/.*",
-    ".*/pyre-check/stubs/.*"
+    ".*/pyre-check/stubs/.*",
+    ".*/tools/integration_tests/.*"
```

Comment:
Why are we excluding this folder?

Reply:
Do we have to check it? Let's check it then :D